pkg/ingester/ingester.go

```diff
@@ -3406,3 +3471,103 @@ type utilizationBasedLimiter interface {
 
 	LimitingReason() string
 }
+
+var ownedSeriesRingOp = ring.WriteNoExtend // Distributor uses WriteNoExtend.
+
+// This is Starting function for ownedSeries service. Service is only started after all TSDBs are opened.
```

Comment:
[nit] With the aim to reduce a bit the `ingester.go` file size, WDYT about moving this function and the following new ones to a dedicated file and dedicated component? It could take in input `Ingester` to get the TSDBs.

Reply:
It's moved to separate file and service now. https://github.com/grafana/mimir/pull/6718/commits/af3af97e7c8791a46f5ae84f94846fa7d3734188